docker-compose.yml

```diff
@@ -79,7 +79,8 @@ services:
         BBSYNC_SYNC_TO_BZ: ${BBSYNC_SYNC_TO_BZ}
         JIRA_TASKMAN_URL: ${JIRA_TASKMAN_URL}
         JIRA_TASKMAN_PROJECT_KEY: ${JIRA_TASKMAN_PROJECT_KEY}
-        HTTPS_TASKMAN_PROXY: ${HTTPS_TASKMAN_PROXY} 
+        HTTPS_TASKMAN_PROXY: ${HTTPS_TASKMAN_PROXY}
```

Comment:
nitpick: I do not understand why is this line readded - they seem the same

Reply:
The original one has a space at the end of the line :)

Reply:
Good catch, I was wondering the same but didn't bother to check lol :)